oak-run-elastic/pom.xml

```diff
@@ -218,6 +218,11 @@
             <version>${project.version}</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tika</groupId>
+            <artifactId>tika-core</artifactId>
+            <version>1.24.1</version>
```

Comment:
we should pick this up from the oak-parent pom properties. https://github.com/apache/jackrabbit-oak/blob/trunk/oak-parent/pom.xml#L62 

<version>${tika.version}</version>

Reply:
+1, we should specify the version only once

Reply:
FWIW; Tika 2.x has been around since July. Minimally, we should update Tika to the latest version from the maintenance branch.

Reply:
Created Jira [OAK-9752](https://issues.apache.org/jira/browse/OAK-9752) to upgrade tika version.